doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1522,144 +1508,108 @@ Managing the local context
 
-      This behaves as :n:`assert (@ident : @type) by exact @term` where
-      :token:`type` is the type of :token:`term`. This is equivalent to using
-      :tacn:`pose proof`. If the head of term is :token:`ident`, the tactic
-      behaves as :tacn:`specialize`.
+      :n:`@ltac_expr3` was not able to prove the provided :n:`@term` or :n:`@one_term`.
 
-      .. exn:: Variable @ident is already declared.
-         :undocumented:
+   .. exn:: Variable @ident is already declared.
+      :undocumented:
 
-.. tacv:: eassert @type as @simple_intropattern by @tactic
-   :name: eassert
+.. tacn:: eassert ( @ident : @term ) {? by @ltac_expr3 }
+          eassert @one_term {? @as_ipat } {? by @ltac_expr3 }
+   :name: eassert; _
 
-   While the different variants of :tacn:`assert` expect that no existential
-   variables are generated by the tactic, :tacn:`eassert` removes this constraint.
-   This lets you avoid specifying the asserted statement completely before starting
-   to prove it.
+   While the different variants of :tacn:`assert` expect that no :term:`existential
+   variables <existential variable>` are generated by the tactic, :tacn:`eassert`
+   removes this constraint.  This lets you avoid specifying the asserted statement
+   completely before starting to prove it.
 
-.. tacv:: pose proof @term {? as @simple_intropattern}
-   :name: pose proof
+.. tacn:: pose proof @term {? @as_ipat }
+          pose proof ( @ident := @term )
+   :name: pose proof; _
 
-   This tactic behaves like :n:`assert @type {? as @simple_intropattern} by exact @term`
+   The first form behaves like :n:`assert @type {? @as_ipat } by exact @term`
    where :token:`type` is the type of :token:`term`. In particular,
-   :n:`pose proof @term as @ident` behaves as :n:`assert (@ident := @term)`
+   :n:`pose proof @term as @ident` behaves like :n:`assert (@ident := @term)`
    and :n:`pose proof @term as @simple_intropattern` is the same as applying the
    :token:`simple_intropattern` to :token:`term`.
 
-.. tacv:: epose proof @term {? as @simple_intropattern}
-   :name: epose proof
-
-   While :tacn:`pose proof` expects that no existential variables are generated by
-   the tactic, :tacn:`epose proof` removes this constraint.
-
-.. tacv:: pose proof (@ident := @term)
-
-   This is an alternative syntax for :n:`assert (@ident := @term)` and
-   :n:`pose proof @term as @ident`, following the model of :n:`pose
-   (@ident := @term)` but dropping the value of :token:`ident`.
-
-.. tacv:: epose proof (@ident := @term)
-
-   This is an alternative syntax for :n:`eassert (@ident := @term)`
-   and :n:`epose proof @term as @ident`, following the model of
-   :n:`epose (@ident := @term)` but dropping the value of
-   :token:`ident`.
+   The second form is equivalent to :n:`assert (@ident := @term)` and
+   :n:`pose proof @term as @ident`, following the model of
+   :n:`pose (@ident := @term)` but dropping the value of :token:`ident`.
```

Comment:
```suggestion
   :n:`pose (@ident := @term)` but dropping the body of :token:`ident`.
```

BTW, another way of saying this is that `pose proof (@ident := @term)` behaves the same as `pose (@ident := @term); clearbody @ident`.

Reply:
I don't see the point of saying it's equivalent to 3 different things (assert, pose proof and pose).  I think saying it's equivalent to assert is sufficient.  Yes?

Reply:
Sure, it is sufficient. But my preference (if we want to mention only one thing) would rather be the `pose ; clearbody` thing since it reflects the link between `pose` and `pose proof`. That's not a strong preference though, so your choice.

Reply:
The description of the first form refers to assert, so IMO best for the second form to also refer to assert.